reporter/src/main/kotlin/reporters/AmazonOssAttributionBuilderReporter.kt

```diff
@@ -104,8 +104,10 @@ class AmazonOssAttributionBuilderReporter : Reporter {
                 // URL passed into the Amazon Oss Attribution Builder needs to be reachable!
                 val pkgUrl = pkg.homepageUrl.takeUnless { it.isEmpty() } ?: "https://github.com/404"
 
-                val licensesWithCopyrights =
-                    input.ortResult.getDetectedLicensesWithCopyrights(pkg.id, input.packageConfigurationProvider)
+                val licenseInfo = input.licenseInfoResolver.resolveLicenseInfo(pkg.id)
+
+                val licensesWithCopyrights = licenseInfo.filter(LicenseView.ONLY_DETECTED)
+                    .associate { it.license.simpleLicense() to it.getCopyrights(omitExcluded = true) }
```

Comment:
Why did you choose to set `omitExcluded = true` here, isn't the default in the previous code `omitExcluded = false`? 

Also, it looks odd to omit excludes for copyrights, but not for their associated licenses, if I read the code correctly.

Reply:
> Why did you choose to set omitExcluded = true here, isn't the default in the previous code omitExcluded = false?

I believe the previous code had `omitExcluded = true`, which is why I explicitly set it.

> Also, it looks odd to omit excludes for copyrights, but not for their associated licenses, if I read the code correctly.

As `licensesWithCopyrights` is only used for obtaining the copyrights, it suffices to look at the value to which `allCopyrights` gets assigned. It is assigned to all `non-excluded` copyrights, same as before this change. Thus the change is non-functional which was my intention.

I prefer to not change functionality in this change.
Would you be happier if I just inlined `licensesWithCopyrights`, in order to not have so `non-excluded` licenses assigned anywhere?

Reply:
I have inlined the variable, so IIUC the issue you raised should be gone.